### PR TITLE
[WBCAMS-549] fix mobile theme for career discovery site

### PR DESCRIPTION
**Pull request by jonathan-longe**
Reviewing this PR from the code alone will be difficult.  I'm happy to share screens in a meeting if that makes it easier.

### CODE REVIEW

File: src/web/modules/custom/workbc_cdq_career_match/js/workbc-cdq-career-match.js [hunk 1]
```diff
@@ -81,30 +81,6 @@
...
-            $(".result-heading h3.vaa").hide();
-            $(".result-heading h3.vaa1").show();
-          }
-          
-        });
-      });
-
       $(once('cdqworkvalues', '.quiz-work-values-wrapper', context)).each(function () {
         $('.work-values-toggle').on('click', function() {
           console.log("toggle");
```
Comment (infojunkie) [inline src/web/modules/custom/workbc_cdq_career_match/js/workbc-cdq-career-match.js:86]: Let's remove this debug output

File: src/web/themes/custom/workbc_cdq/scss/_components.scss [hunk 2]
```diff
@@ -1,1683 +1,3976 @@
...
+
+  @media all and (max-width: 767.98px){
+    margin: 0 0;
   }
+}
 
+// talk to Ray about removing the offending content.
+h2.cdq-quiz-title, .webform-submission-information {
+  display: none
+}
```
Comment (jonathan-longe) [inline src/web/themes/custom/workbc_cdq/scss/_components.scss:3976]: Ray, I assume hiding this content is okay with you?

**Comment by infojunkie** [comment]
Thanks I'll try it locally

File: src/web/themes/custom/workbc_cdq/scss/_components.scss (hunk 2 above)
Comment (infojunkie) [reply src/web/themes/custom/workbc_cdq/scss/_components.scss:3976]: @rjkuyvenhoven

**Comment by rjkuyvenhoven** [comment]
I've noticed a couple things

Desktop
Modify Quiz Answers page - sub title text should not be black
Compare Careers page - the page content is no longer centered

Mobile
Quiz Results page - does not match original styling and functionality

**Comment by jonathan-longe** [comment]
Homepage at desktop resolution shown below.  Apologies for the PDFs but this seems like the best way of showing full-length pages.
[Home_Desktop.pdf](https://github.com/user-attachments/files/16686617/Home_Desktop.pdf)

**Comment by jonathan-longe** [comment]
Quiz pages shown below:
[Quiz_Desktop.pdf](https://github.com/user-attachments/files/16686631/Quiz_Desktop.pdf)
[Quiz_Iphone.pdf](https://github.com/user-attachments/files/16686632/Quiz_Iphone.pdf)

**Comment by jonathan-longe** [comment]
Results pages shown below:
[Results_Desktop.pdf](https://github.com/user-attachments/files/16686636/Results_Desktop.pdf)
[Results_Iphone.pdf](https://github.com/user-attachments/files/16686637/Results_Iphone.pdf)

**Comment by jonathan-longe** [comment]
Comparison pages shown below:
[Compare_Desktop.pdf](https://github.com/user-attachments/files/16686642/Compare_Desktop.pdf)
[Compare_Iphone.pdf](https://github.com/user-attachments/files/16686644/Compare_Iphone.pdf)

**Comment by jonathan-longe** [comment]
Thanks for the feedback @rjkuyvenhoven !!!  I'll respond below:

> Desktop
Modify Quiz Answers page - sub title text should not be black 

I couldn't find black sub-title text, can you send me a link?  

> Compare Careers page - the page content is no longer centered

Good catch, I've fixed this.

> Mobile
 Quiz Results page - does not match original styling and functionality

Yes, that's correct.  The career profiles aren't currently shown in a carousel, but  I think this is consistent with the way we show other card-type content in mobile.  If our client doesn't like it, I can implement a carousel.

**Comment by rjkuyvenhoven** [comment]
> Thanks for the feedback @rjkuyvenhoven !!! I'll respond below:
> 
> > Desktop
> > Modify Quiz Answers page - sub title text should not be black
> 
> I couldn't find black sub-title text, can you send me a link?

On the front page click the "Modify Your Answers" link for a completed quiz.  Alternatively, click the "Modify Your Answers" button on the bottom right of the results page.

**Comment by rjkuyvenhoven** [comment]
@jonathan-longe Here is a screenshot of the Work Preferences results page from PROD

Note: the % Match section is a slideshow and the Career Match section excludes Annual Salary and Training, Education, Experience & Responsibilities columns from the table.

Work-Preferences-Quiz-Results-Career-Discovery-Quizzes

**Comment by jonathan-longe** [comment]
> Note: ... the Career Match section excludes Annual Salary and Training, Education, Experience & Responsibilities columns from the table.

Thanks @rjkuyvenhoven , I hadn't noticed that difference.  I'll fix it!

**Pull request merged**